log.go

```diff
@@ -155,6 +173,10 @@ func (l Logger) Output(w io.Writer) Logger {
 	l2 := New(w)
 	l2.level = l.level
 	l2.sampler = l.sampler
+	if len(l.hooks) > 0 {
+		l2.hooks = make([]Hook, len(l.hooks), cap(l.hooks))
```

Comment:
You don't need to set the cap if equal to the size.

Reply:
I'm just appending now.